app/ide-desktop/common/src/text/english.json

```diff
@@ -698,6 +733,7 @@
   "organizationSettingsTab": "Organization",
   "organizationSettingsSection": "Organization",
   "organizationNameSettingsInput": "Organization display name",
+  "organizationNameSettingsInputDescription": "The name should not be longer that $0 characters",
```

Comment:
`longer that` -> `longer than`.
also optionally `The name` -> `The organization's name`?

Reply:
> also optionally The name -> The organization's name?

Not sure. In that case, I think, it's clear enough what we mean by `name`
![CleanShot 2024-07-28 at 19 57 15@2x](https://github.com/user-attachments/assets/cb2a5108-0d6c-41c3-8020-a6b7a7a42fde)